src/dotnet/commands/dotnet-migrate/MigrateCommand.cs

```diff
@@ -281,17 +281,25 @@ private string GetReportSummary(MigrationReport migrationReport)
             StringBuilder sb = new StringBuilder();
 
             sb.AppendLine("Summary");
-            sb.AppendLine($"Total Projects: {migrationReport.MigratedProjectsCount}");
-            sb.AppendLine($"Succeeded Projects: {migrationReport.SucceededProjectsCount}");
-            sb.AppendLine($"Failed Projects: {migrationReport.FailedProjectsCount}");
+            sb.AppendLine(
```

Comment:
Also, not in this PR. :)  I was planning to this up as part of the resx codegen.